cmd/wallet-web/src/pages/Store.vue

```diff
@@ -225,7 +224,7 @@ export default {
         });
     },
     vaultSelected: function (e) {
```

Comment:
This method should be called `setSelectedVault`, because what does is set `this.selectedVault`